apps/web/helpers/cover-image.helper.ts

```
@@ -0,0 +1,271 @@
+  IMAGE_21: "/images/image_21.jpg",
+  IMAGE_22: "/images/image_22.jpg",
+  IMAGE_23: "/images/image_23.jpg",
+  IMAGE_24: "/images/image_24.jpg",
+  IMAGE_25: "/images/image_25.jpg",
+  IMAGE_26: "/images/image_26.jpg",
+  IMAGE_27: "/images/image_27.jpg",
+  IMAGE_28: "/images/image_28.jpg",
+  IMAGE_29: "/images/image_29.jpg",
+} as const;
+
+export const DEFAULT_COVER_IMAGE_URL = STATIC_COVER_IMAGES.IMAGE_1;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)